### PR TITLE
feat: bucket data can be added to environment variables

### PR DESCRIPTION
**Pull request by peterpaulo-azion**
WHAT
Include the possibility that the AWS bucket configuration can be included by environment variables as well.

### CODE REVIEW

**Comment by peterpaulo-azion** [comment]
- - -
   ***I have read the CLA Document and I hereby sign the CLA***
   - - -

**Comment by peterpaulo-azion** [comment]
recheck

File: README.md [hunk 1]
```diff
@@ -52,9 +52,20 @@ git clone https://github.com/aziontech/cells-site-template
...
+You can alternatively enter the bucket data in the `environment variables`:
+
+```
+AWS_DEFAULT_BUCKET_NAME=bucket-name
+AWS_DEFAULT_BUCKET_REGION=sa-east-1
+AWS_DEFAULT_BUCKET_PATH=path
+```
+
+The access credentials to the bucket will be stored in the code sent to Azion and the Edge Function will use them to retrieve the necessary pages stored during the publish step.
+Similarly, you will need a [Personal Token](https://www.azion.com/pt-br/documentacao/produtos/gestao-de-contas/personal-tokens) to deploy your project at Azion and you can either set those in the `azion.json` file or define the environment variable `AZION_TOKEN`. The Azion credentials are not stored in the built code.
```
Comment (PatrickMenoti) [inline README.md:66]: You can export it as an environment varialbe, or use azioncli configure -t <token> command.

**Review by PatrickMenoti — CHANGES_REQUESTED** [review]

**Review by PatrickMenoti — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -52,9 +52,20 @@ git clone https://github.com/aziontech/cells-site-template
 
 ## Build
 
-You will need a bucket in AWS S3, its access key and its secret access key. You can set the credentials at the `azion.json` file inside the project or define the environment variables `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`. The access credentials to the bucket will be stored in the code sent to Azion and the Edge Function will use them to retrieve the necessary pages stored during the publish step.
-Similarly, you will need credentials to deploy your project at Azion and you can either set those in the `azion.json` file or define the environment variable `AZION_TOKEN`. The Azion credentials are not stored in the built code.
+You will need a bucket in AWS S3, its access key and its secret access key. You can set the credentials at the `azion.json` file inside the project or define the environment variables `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.
```
[the remark this answers is not in the document]
Comment (MagnunAVFAzion) [reply README.md:55]: azion-framework-adapter is not only used in cli context, it is also used in vscode plugin. All operations in the jamstack build/publish context must remain in this repository.

File: README.md (hunk 1 above)
Comment (MagnunAVFAzion) [reply README.md:66]: azion-framework-adapter is not only used in cli context, it is also used in vscode plugin. All operations in the jamstack build/publish context must remain in this repository.

**Review by MagnunAVFAzion — CHANGES_REQUESTED** [review]
New azion-framework-adapter use cases are added in this PR (env vars). We need tests (mainly unit tests) for these cases.

**Comment by MagnunAVFAzion** [comment]
moved to #34

**Pull request closed without merging**